tasks/packager/tv-webos.js

```diff
@@ -186,6 +192,38 @@ function askUserData(cordovaConf, callback, versionOnly, userData) {
                 return typeof(input) === 'string' ? true : 'invalid iconpath';
             }
         }, {
+            type: 'input',
+            name: 'bgimage',
+            message: 'Background image path (default is \'img/logo.png\')',
+            default: 'img/logo.png',
+            validate: function(input) {
+                return typeof(input) === 'string' ? true : 'invalid path';
+            }
+        }, {
+            type: 'input',
+            name: 'splashbackground',
+            message: 'Splash screen background path (default is \'img/logo.png\')',
+            default: 'img/logo.png',
+            validate: function(input) {
+                return typeof(input) === 'string' ? true : 'invalid path';
+            }
+        }, {
+            type: 'input',
+            name: 'bgcolor',
+            message: 'Background color for app',
+            default: 'red',
```

Comment:
Could you change default color to white or black (achromatic color)?
I think it's better to have an inconspicuous color.